contracts/IERC721M.sol

```diff
@@ -52,4 +54,43 @@ interface IERC721M {
     event SetBaseURI(string baseURI);
     event PermanentBaseURI(string baseURI);
     event Withdraw(uint256 value);
+
+    function getCosigner() external view returns (address);
+
+    function getCrossmintAddress() external view returns (address);
+
+    function getNumberStages() external view returns (uint256);
+
+    function getGlobalWalletLimit() external view returns (uint256);
+
+    function getMaxMintableSupply() external view returns (uint256);
+
+    function getMintable() external view returns (bool);
+
+    function totalMintedByAddress(address a) external view returns (uint256);
+
+    function getTokenURISuffix() external view returns (string memory);
+
+    function getStageInfo(uint256 index)
```

Comment:
Do we need uint256 for stage index?

Reply:
I believe one storage slot is 32 bytes (256 bits), so anything smaller will still require 1 slot allocated https://docs.soliditylang.org/en/v0.8.15/internals/layout_in_storage.html